playground/miden-wasm/src/transaction.rs

```diff
@@ -1,285 +0,0 @@
-use super::ExecutionProof;
```

Comment:
The whole file has been deleted, the tx proving is handled directly in the lib.rs in this new version right?

Reply:
at the moment, we don't prove transactions in the playground. We soon will be, but then we use the whole client.